README.md

```diff
@@ -0,0 +1,93 @@
+
+    ```
+    metrics = find_metrics('https://prom-1.dj-test.dev.gds-reliability.engineering/','prometheus')
+    ```
+
+    This is the section you need to edit. You also need to edit the variable in the locust exporter file.
+
+    ```
+    ENV PROM_SERVER='https://prom-1.dj-test.dev.gds-reliability.engineering/'
+    ```
+
+    c. ### IF YOU NEED AUTH EDIT THE LOCUST FILE USING [THESE INSTRUCTIONS](http://docs.python-requests.org/en/master/user/authentication/#basic-authentication)
+
+    d. build the initial containers with the following commands:
+        
+
+        docker build -t local/locustserver .
+        cd locustexporter
+        docker build -t local/locustexporter .
+
+
+    e. Please also make sure that you configure prometheus to scrape the prometheus that is the system under test. This can be done by adding the target prometheus server to the scrape config defined here: 
+    
+        {project_root}/config/prometheus-config.yaml
+
+    f. If all is well you can configure the enviroment. In order to do that we run a docker compose command that should bring everything up as expected. Run this command as root since we have seen some issues with it.
+
+        sudo docker-compose -f compose-env.yaml
+
+    g. All should of gone well. You can verify this by running the following command and receiving similar output.
+
+        docker ps 
+        root@ip-10-0-101-220:/home/ubuntu# docker ps
+        CONTAINER ID        IMAGE                    COMMAND                  CREATED             STATUS              PORTS                    NAMES
+        9e10a00d6b47        local/locustexporter     "python /locust_ex..."   23 seconds ago      Up 21 seconds       0.0.0.0:4455->4455/tcp   observer-load-test_locustexporter_1
+        114aded15c69        grafana/grafana:latest   "/run.sh"                24 seconds ago      Up 23 seconds       0.0.0.0:3000->3000/tcp   grafana
+        cbb40d3b47da        local/locustserver       "/start.sh"              24 seconds ago      Up 23 seconds       0.0.0.0:8089->8089/tcp   observer-load-test_locustserver_1
+
+
+## Operating the load test
+
+### AWS
+
+1. You have to either open the firewall to allow all traffic from your IP or configrue access to the required ports
+
+    a. Access ports reuqired:
+
+        grafana:3000
+        prometheus:9090
+        locust:8089
+
+2. Access promtheus in order to verify all targets including locust.
+
+![Promtheus targets](images/targets.png?raw=true "Prometheus targets")
+
+3. Access grafana in order to determine that all grafana's are avalible and that they have been loaded properly (defautl password: admin:secret)
+
+![Prometheus dashboard](images/grafana.png?raw=true "Prometheus Grafana")
+
+
+![Locust dashboard](images/locust.png?raw=true "Locust")
+
+2. Access locust and set the value's that you need in order to evaluate your service. The more users the better.
```

Comment:
Could you maybe expand on this or point to some docs about what the inputs are such as swarm rate and users etc

Reply:
Peeps will have to read the docs for that which I have referenced via a link